xicam/spectral/stages/__init__.py

```diff
@@ -42,10 +44,16 @@ def __init__(self):
         super(SpectralBase, self).__init__()
 
     def treatment_kwargs(self, workflow):
+        intent_name = ['hyperspectral_data', 'ptychography data', 'IR maps']
```

Comment:
@ronpandolfi what's the best way to attribute scientific meaning through intents?